processor/transaction/processor.go

```diff
@@ -22,18 +23,28 @@ var (
 )
 
 const (
-	Endpoint      = "/v1/transactions"
-	processorName = "transaction"
+	BackendEndpoint  = "/v1/transactions"
+	FrontendEndpoint = "/v1/client-side/transactions"
+	processorName    = "transaction"
 )
 
-func NewProcessor() pr.Processor {
+func NewBackendProcessor() pr.Processor {
 	return &processor{
 		schema: pr.CreateSchema(transactionSchema, processorName),
+		typ:    pr.Backend,
+	}
+}
+
+func NewFrontendProcessor() pr.Processor {
+	return &processor{
+		schema: pr.CreateSchema(transactionSchema, processorName),
+		typ:    pr.Frontend,
 	}
 }
 
 type processor struct {
 	schema *jsonschema.Schema
+	typ    int
```

Comment:
`type` ?

Reply:
`type` is a reserved word, can't be a variable name

Reply:
ah right. How about `processorType` ? we already have `processorName` for example

Reply:
is that really necessary? i'm usually told to use short and concise (ie idiomatic) names, and seems you are pushing the opposite way. `typ` is a var name extensively used in Go's source code itself. 
I don't mind either way, but we should align on this to avoid having the same discussion over and over.

That variable is only used in a getter:
```
func (p *processor) Type() int {
	return p.typ
}
```
Isn't it clear enough? i'm calling this as `processor.Type()`

Reply:
+1 on calling it `typ`. 
It is only used internally, and as @jalvz pointed out `typ` is heavuly used in the `go` source code itself.